<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatSystem</class>
 <widget class="QMainWindow" name="ChatSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="userTableWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>-20</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>主机名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP地址</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="sendToolBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/send.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="italicToolBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/italic.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>561</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="clearToolBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/clear.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="colorToolBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/color.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="userNumLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>在线人数：0人</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="underlineToolBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/under.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="saveToolBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/save.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="boldToolBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/tools/images/bold.png)</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
